issue_processing/jira_issue_processing.py

```diff
@@ -0,0 +1,401 @@
+    """Load persons from disk.
+
+    :param source_folder: the folder where to find .csv-file
+    :return: the loaded person data
+    """
+
+    srcfile = os.path.join(source_folder, "jira-comment-authors-with-email.csv")
+    log.devinfo("Loading person csv from file '{}'...".format(srcfile))
+
+     # check if file exists and exit early if not
+    if not os.path.exists(srcfile):
+        log.info("Person file '{}' does not exist! Exiting early...".format(srcfile))
+        sys.exit(-1)
+
+
+    person_data = list()
+     #with open(srcfile, 'r') as person_file:
+    with open(srcfile, 'r') as f:
+         person_data = csv.DictReader(f, delimiter=',', skipinitialspace=True)
+
+    return person_data
+
+
+def parse_csv(person_date):
+    for row in person_date:
+        print(row['AuthorID'], row['userEmail'])
+
+	return persons
+
+def parse_xml(issue_data):
+    """Parse issues from the xml-data
+
+        :param issue_data: xml-data
+        :return: the loaded issue data
+        """
+
+    log.devinfo("Parse issues...")
+    issues = list()
+
+    log.debug("Number of files:" + str(len(issue_data)))
+    for issue_file in issue_data:
+        issuelist = issue_file.getElementsByTagName('item')
+        # re-process all issues
+        log.debug("Number of issues:" + str(len(issuelist)))
+        for issue_x in issuelist:
+            # temporary container for references
+            comments = list()
+            issue = dict()
+
+            #parse values form xml
+            #add issue values to the issue
+            key = issue_x.getElementsByTagName('key')[0]
+            issue['id'] = key.attributes['id'].value
+            issue['externalId'] = key.firstChild.data
+
+            created = issue_x.getElementsByTagName('created')[0]
+            createDate = created.firstChild.data
+            issue['creationDate'] = datetime.strptime(createDate, '%a, %d %b %Y %H:%M:%S +0000')
+
+            resolved = issue_x.getElementsByTagName('resolved')
+            issue['resolveDate'] = ""
+            if (len(resolved) > 0) and (not resolved[0] is None) :
+                resolveDate = resolved[0].firstChild.data
+                issue['resolveDate'] = datetime.strptime(resolveDate, '%a, %d %b %Y %H:%M:%S +0000')
+
+            link = issue_x.getElementsByTagName('link')[0]
+            issue['url'] = link.firstChild.data
+
+            type = issue_x.getElementsByTagName('type')[0]
+            issue['type'] = type.firstChild.data
+
+            status = issue_x.getElementsByTagName('status')[0]
+            issue['state'] = status.firstChild.data
+
+            project = issue_x.getElementsByTagName('project')[0]
+            issue['projectId'] = project.attributes['id'].value
+
+            assignee = issue_x.getElementsByTagName('reporter')[0]
+            user = dict()
+            user["username"] = assignee.attributes['username'].value
+            user["name"] = assignee.attributes['username'].value	##assignee.firstChild.data

```

Comment:
I think, it is not necessary, to use the id-manager here, because the id-manager was already used during the conway-codeface-analysis.

Reply:
Just leaving this a comment, although we already talked about that: We still need the name and the e-mail address from the Codeface database to be able to align the issue data with data from other sources (e.g., mailing-list data). 
So, we need to parse the CSV file after all, read the authors and their stored e-mail addresses from the database and use those instead of the username here.

Reply:
We have a problem with the CSV file. It doesn't include all authors. I think, they are all reporters of a issue.